marc_to_solr/translation_maps/holding_library.rb.tmpl

```diff
@@ -1,26 +1,25 @@
-{ "recap$gp" => "Firestone Library",
-  "recap$pb" => "Firestone Library",
-  "recap$pf" => "Firestone Library",
-  "recap$pg" => "Rare Books and Special Collections",
-  "recap$ph" => "Mudd Manuscript Library",
-  "recap$pj" => "Marquand Library",
-  "recap$pk" => "Mendel Music Library",
-  "recap$pl" => "East Asian Library",
-  "recap$pm" => "Stokes Library",
-  "recap$pn" => "Lewis Library",
-  "recap$pq" => "Harold P. Furth Plasma Physics Library",
-  "recap$ps" => "Lewis Library",
-  "recap$pt" => "Engineering Library",
-  "recap$pw" => "Architecture Library",
-  "recap$pz" => "Marquand Library",
-  "recap$qk" => "Mendel Music Library",
-  "recap$ql" => "East Asian Library",
-  "recap$qv" => "Video Library",
-  "recap$xc" => "Rare Books and Special Collections",
-  "recap$xg" => "Rare Books and Special Collections",
-  "recap$xm" => "Rare Books and Special Collections",
-  "recap$xn" => "Rare Books and Special Collections",
-  "recap$xp" => "Rare Books and Special Collections",
-  "recap$xr" => "Rare Books and Special Collections",
-  "recap$xx" => "Rare Books and Special Collections",
-  "rare$warf" => "Rare Books and Special Collections" }
+{"recap$gp"=>"ReCAP - rcpgp RECAP Gov Docs",
+ "recap$pb"=>"ReCAP - rcppb RECAP Firestone Use",
+ "recap$pf"=>"ReCAP - rcppf RECAP Microforms",
+ "recap$pg"=>"ReCAP - rcppg RECAP RBSC",
+ "recap$ph"=>"ReCAP - rcpph RECAP Mudd Lib.",
+ "recap$pj"=>"ReCAP - rcppj RECAP Marq. Lib.",
+ "recap$pk"=>"ReCAP - rcppk RECAP Music Lib.",
+ "recap$pl"=>"ReCAP - rcppl RECAP East Asian",
+ "recap$pm"=>"ReCAP - rcppm RECAP Stokes Lib",
+ "recap$pn"=>"ReCAP - rcppn RECAP Lewis Library",
+ "recap$pq"=>"ReCAP - rcppq RECAP PlasmaPhysics",
+ "recap$ps"=>"ReCAP - rcpps RECAP Lewis Rare",
+ "recap$pt"=>"ReCAP - rcppt RECAP Tech Reports",
+ "recap$pw"=>"ReCAP - rcppw RECAP Architecture",
+ "recap$pz"=>"ReCAP - rcppz RECAP Marquand Rare",
+ "recap$qk"=>"ReCAP - rcpqk RECAP Music Lib.",
+ "recap$ql"=>"ReCAP - rcpql RECAP EA Microforms",
+ "recap$qv"=>"ReCAP - rcpqv RECAP Video Coll.",
+ "recap$xc"=>"ReCAP - rcpxc RECAP Cotsen Lib.",
+ "recap$xg"=>"ReCAP - rcpxg RECAP Graphic Arts",
+ "recap$xm"=>"ReCAP - rcpxm RECAP Manuscripts",
+ "recap$xn"=>"ReCAP - rcpxn RECAP Numismatics",
+ "recap$xp"=>"ReCAP - rcpxp RECAP Historic Maps",
+ "recap$xr"=>"ReCAP - rcpxr RECAP Rare Books",
+ "recap$xx"=>"ReCAP - rcpxx RECAP Special"}
```

Comment:
These are the holding libraries that I get from Alma. Will we update the names in Alma? Should I proceed and add a mapping based on the old file? @mzelesky

Reply:
Well I dont think it's necessary to add a mapping for the names right now. When they will updated in Alma we will pull them again. @mzelesky can you confirm?